koap/src/jsMain/kotlin/App.kt

```diff
@@ -0,0 +1,361 @@
+import com.juul.koap.Message
```

Comment:
This file doesn't belong in the main `koap` module.
The `koap` module is the core library that we'll publish and shouldn't contain the sample app.
Instead, this should live in a separate module (that depends on the `koap` module).